src/components/calcite-tooltip/resources.ts

```diff
@@ -0,0 +1,11 @@
+import Popper from "popper.js";
+
+export const CSS = {
+  container: "container",
```

Comment:
do we want to use tooltip-container, tooltip-container-* ?

Reply:
Doesn't matter since its shadow dom. We could change it if you like :)